src/orion/algo/evolution_es.py

```diff
@@ -190,6 +190,10 @@ def get_candidates(self, rung_id):
         blue_team = self.eves.rng.choice(diff_list, self.eves.nums_comp_pairs,
                                          replace=False)
 
+        return rung, population_range, red_team, blue_team
+
+    def _get_mutated_population(self, red_team, blue_team):
```

Comment:
```suggestion
    def _mutate_population(self, red_team, blue_team):
```
Since it is modifying the population in-place.

Reply:
Done.